master/internal/task/postgres_allocations.go

```diff
@@ -0,0 +1,19 @@
+package task
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// AddAllocationAcceleratorData stores acceleration data for an allocation.
+func AddAllocationAcceleratorData(ctx context.Context, accData model.AcceleratorData,
```

Comment:
i don't think this function needs to be exported

Reply:
It needs to be exported for the integration tests

Reply:
I write tests in package `abc_test` to test the public interface of a package but I wouldn't add a function to the public interface just to test it, I would put the tests in `abc` instead.

Reply:
The idea is to keep the public API of a package limited to exactly what is necessary.